scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1252 @@
+
+    with open(os.path.join(temp_dir, "requirements.txt"), 'w') as f:
+        f.write(requirements)
+    print("  ✅ Created requirements.txt")
+
+    # Validate critical files exist (avoid common pitfalls)
+    print("\n🔍 VALIDATING MODEL FILES...")
+    critical_files = ['config.json', 'tokenizer.json', 'tokenizer_config.json']
+    missing_files = []
+
+    for file in critical_files:
+        file_path = os.path.join(temp_dir, file)
```

Ruff (0.12.2)

`os.path.join()` should be replaced by `Path` with `/` operator

(PTH118)